csharp/ql/src/semmle/code/csharp/frameworks/Format.qll

```diff
@@ -5,6 +5,7 @@
 import csharp
 private import semmle.code.csharp.frameworks.System
 private import semmle.code.csharp.frameworks.system.Text
+private import semmle.code.csharp.dataflow.DataFlow2
```

Comment:
The slight weakness of this API is that a user has no way of knowing that there could be a conflict between `Format.qll` and `DataFlow2.qll`. Would it make sense to use `DataFlow5` here to minimize this?